samples/interactive-tutorials/user_import_data_to_catalog.sh

```diff
@@ -14,6 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# activate the python virtual env
+source ~/cloudshell_open/myenv/bin/activate
```

Comment:
`~/cloudshell_open` doesn't exist in my setup. Do I need to create the directory `cloudshell_open`?

Reply:
we have expect that the tutorials will bde run in the Cloud Shell enwironment, and the ~/cloudshell_open it is the standart workspase root directory  in CSh, which is created by default. If the code will be clonned somewhere outside the ~/cloudshell_open directory, some of Cloud Shell Tutorias features will not work (like open file and spotlight some text). We do not expect the code samples will be run in some other environments